config/locales/cy.yml

```diff
@@ -267,6 +267,13 @@ cy:
         few:
         many:
         other: Areithiau
+      standard:
+        zero:
+        one: Safon
+        two:
+        few:
+        many:
```

Comment:
Can we remove the empty `zero`/`two`/`few`/`many` properties, like in the English version?

Reply:
The [pluralisation rules for Welsh](https://github.com/svenfuchs/rails-i18n/blob/7a273036d94373477ec1abc2be154ccdf0a252f7/rails/pluralization/cy.rb) require us to have a `zero`, `two`, `few` and `many` value.  We weren't provided with translations for these, so the empty values should help us identify outstanding translations when we do [this piece of work](https://trello.com/c/ifbyj6pJ/1831-identify-keys-in-locale-files-that-require-translations).  I've also done the same for other languages that don't have a `one-other` pluralisation rule (e.g. Croatian).